src/integration-tests/apps/webservices/META-INF/application.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
copyright

Reply:
this file should not be in the PR, removed